packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -403,7 +410,8 @@ export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
     }
 
     protected renderStars(): React.ReactNode {
-        const rating = this.props.extension.averageRating;
+        const rating = this.props.extension.averageRating || 0;
+
         if (typeof rating !== 'number') {
```

Comment:
then rating is not number?

Reply:
When `rating` is null, we assign 0 to it.

Reply:
@akosyakov  Since we already check if `rating` is `undefined` here:
https://github.com/eclipse-theia/theia/blob/1ac32c57cb569e0883312632924c512a6ec67fac/packages/vsx-registry/src/browser/vsx-extension.tsx#L380

Reply:
I will remove this part.